File: clients/javascript/src/models/streamReceiveOut.ts
// this file is @generated
import {
    type StreamMsgOut,
    StreamMsgOutSerializer,
} from './streamMsgOut';





export interface StreamReceiveOut {
    msgs: StreamMsgOut[];
}

export const StreamReceiveOutSerializer = {
    _fromJsonObject(object: any): StreamReceiveOut {
        return {
            msgs: object['msgs'].map((item: StreamMsgOut) => StreamMsgOutSerializer._fromJsonObject(item)),
            };
    },

    _toJsonObject(self: StreamReceiveOut): any {
        return {
            'msgs': self.msgs.map((item) => StreamMsgOutSerializer._toJsonObject(item)),
            };
    }
}

File: clients/rust/src/models/stream_receive_out.rs
// this file is @generated
use serde::{Deserialize, Serialize};

use super::stream_msg_out::StreamMsgOut;

#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct StreamReceiveOut {
    pub msgs: Vec<StreamMsgOut>,
}

impl StreamReceiveOut {
    pub fn new(msgs: Vec<StreamMsgOut>) -> Self {
        Self { msgs }
    }
}
